src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.project;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Project's deadline in TaskHub.
+ * Guarantees: immutable; is always valid
+ */
+public class Deadline {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Project deadline should be a valid date in the dd/MM/yyyy format.\n Example: 17/02/2009";
```

Comment:
Same as before - do you think we could use dashes? And also do you think we could allow them to enter timings for the deadlines to be more specific?

Reply:
I have changed from slashes to hyphens. Regarding allowing them to enter timings for the deadlines, I think that could be an enhancement in the future. We can just keep it to date for now.